File: JavaScript_Courses/JavaScript_Basics/JS_BP_2022_10/02_Conditional_Statements_Exercise/02_Bonus_Score.js
function bonusScore(input) {
    let startingPoints = Number(input[0]);
    
    let bonusPoints = 0;

    if (startingPoints <= 100) {
        bonusPoints += 5;
    } else if (startingPoints > 1000) {
        bonusPoints += startingPoints * 0.1;
    } else {
        bonusPoints += startingPoints * 0.2;
    }

    if (startingPoints % 2 == 0) {
        bonusPoints += 1;
    }

    if (startingPoints % 10 == 5) {
        bonusPoints += 2;
    }

    let finalPoints = startingPoints + bonusPoints;

    console.log(bonusPoints);
    console.log(finalPoints);
}

// bonusScore(["175"]);

File: Python_Courses/Python_Basics/Python_PB_2024_10/02_Conditional_Statements_Exercise/02_Bonus_Score.py
starting_points = int(input())

bonus_points = 0
if starting_points <= 100:
    bonus_points = 5
elif starting_points <= 1000:
    bonus_points = starting_points * 0.20
else:
    bonus_points = starting_points * 0.10

addition_bonus_points = 0
if starting_points % 2 == 0:
    addition_bonus_points = 1

if starting_points % 10 == 5:
    addition_bonus_points = 2

total_bonus = bonus_points + addition_bonus_points
total_result = starting_points + total_bonus

print(total_bonus)
print(total_result)
